packages/code-snippet-block/README.md

```diff
@@ -0,0 +1,4 @@
+# Code Snippet Block
+
+Code Snippet Block is text editor specialized for editing code based on CodeMirror v6 with support for multiple languages and themes.
```

Comment:
nice description 👍 

nit: 
Code Snippet Block is **a** text editor specialized for editing code based on CodeMirror v6 with support for multiple languages and themes.
**It provides** additional customization for padding, border and few other options **in** the settings.